fixed typo in pom.xml

arvindkrishnakumar-okta opened the pull request:
<!-- 
Before creating an issue or submitting a PR, please check that your issue is not already fixed in the latest stable version and that a similar issue or PR is not reported already (also check closed issues).
-->

<!--
Please help us process GitHub Issues faster by providing the following information.

Note: If you have a question about your entire application or use case, please post it on the Okta Developer Forum (https://devforum.okta.com) instead. For urgent issues contact support@okta.com. Issues in this repository are reserved for bug reports and feature requests.
-->

## Issue(s)
<!-- Reference any existing issue(s) here. -->

## Description
<!-- Add a brief description of the issue. -->

## Category
<!-- If possible, commit unit tests separately from the implementation to simplify validation. -->
- [ ] Bugfix
- [ ] Enhancement
- [ ] New Feature
- [ ] Configuration Change
- [ ] Versioning Change
- [ ] Unit Test(s)
- [ ] Documentation

sergiishamrai-okta reviewed (changes requested):
I believe this should be also updated 
https://github.com/okta/okta-sdk-java/blob/491027a1d86b99dc56f5cd388963a62fa68152bd/httpclients/okhttp/pom.xml#L38

arvindkrishnakumar-okta commented:
> I believe this should be also updated
> https://github.com/okta/okta-sdk-java/blob/fix_typo_in_pom/httpclients/okhttp/pom.xml#L38

yes, noticed it. fixed.

sergiishamrai-okta reviewed (approved):
looks good
